content/en/security/cloud_security_management/setup/agentless_scanning/azure_resource_manager.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Setting up Agentless Scanning using Azure Resource Manager
+further_reading:
+  - link: "/security/cloud_security_management/agentless_scanning"
+    tag: "Documentation"
+    text: "Cloud Security Management Agentless Scanning"
+  - link: "/security/cloud_security_management/setup/agentless_scanning/terraform"
+    tag: "Documentation"
+    text: "Setting up Agentless Scanning using Terraform"
+aliases:
+  - /security/cloud_security_management/guide/agentless_azure_integration
+---
+
+If you've already [set up Cloud Security Management][3] and want to add a new Azure subscription or enable [Agentless Scanning][1] on an existing integrated Azure subscription, you can use either [Terraform][2] or Azure Resource Manager. This article provides detailed instructions for the Azure Resource Manager approach.
+
+<div class="alert alert-info">If you want to enable Agentless Scanning for an Azure subscription that is not already integrated in Datadog, start by <a href="/integrations/guide/azure-manual-setup/?tab=azurecli">integrating the Azure subscription</a> then enable Agentless Scanning by following the tutorial below.</div>
+
+<div class="alert alert-warning">Running Agentless scanners incurs additional costs. To optimize these costs while still ensuring reliable 12-hour scans, Datadog recommends setting up <a href="/security/cloud_security_management/setup/agentless_scanning/terraform/">Agentless Scanning with Terraform</a> as the default template.</div>
+
+{{< tabs >}}
+{{% tab "Existing Azure subscription" %}}
```

Comment:
Related to my question above re: the note and adding a new Azure subscription, I think we should either a.) Add a tab for new Azure subscription, or b.) remove the tabs entirely. I'm not sure it makes much sense to have only a single tab.

Reply:
I created a second tab.